crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -123,11 +123,11 @@ def repocheck
           } unless os_available
 
           cloud_available = repo_version_available?(products, "suse-openstack-cloud", "8")
-          ret[:cloud] = {
+          ret[:openstack] = {
             available: cloud_available,
             repos: {}
           }
-          ret[:cloud][:repos][admin_architecture.to_sym] = {
+          ret[:openstack][:repos][admin_architecture.to_sym] = {
             missing: ["SUSE OpenStack Cloud 8"]
```

Comment:
That backport is intentionally doing more, that's why it has 2 commits.

I do not think the other one is needed for master - this part is usable for 7-8 upgrade only and by that time we should not need to fake the responses, but have the backend repocheck method really return names of repositories that are missing